scripts/tokenize_data.sh

```diff
@@ -1,3 +1,4 @@
 python3 \
-    ../../tokenize_data.py \
-    ../../configs/user_configs/<YOUR_CONFIG_HERE>.yaml
+    ../../src/tokenize_data.py \
+    ../../configs/user_configs/<YOUR_CONFIG_HERE>.yaml \
+    <YOUR_SPLIT_HERE> # train, validation, or test. You can start a job for each split.
```

Comment:
Lets indicate the dtype for this in the comment (just add string quotes around each option)

Reply:
Done!